scripts/buildtools/commands/verify-package.js

```diff
@@ -3,8 +3,8 @@ const { readFileSync } = require('fs');
 
 const KB = 1024;
 const MB = 1024 * 1024;
-const EXPECTED_PACKAGE_SIZE = 49 * MB;
-const EXPECTED_PACKAGE_FILES = 20000;
+const EXPECTED_PACKAGE_SIZE = 23.5 * MB;
+const EXPECTED_PACKAGE_FILES = 3760;
```

Comment:
Deleting `node_modules/` in the "prepack" step (above) brought our size and file count way back down to Earth. It used to be closer to this range before the v3 code was brought into the output bundle: https://github.com/okta/okta-signin-widget/blob/eed162d9a1d060e9864bd06b132dfab92f3273fc/scripts/buildtools/commands/verify-package.js#L6-L7